test/evergreen.yml

```diff
@@ -1619,8 +1683,8 @@ buildvariants:
   run_on:
   - macos-1012
   expansions:
-    smp_command: -j $(sysctl -n hw.logicalcpu)
     configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb"
+    smp_command: -j $(sysctl -n hw.logicalcpu)
```

Comment:
Any special reason to move this line?

Reply:
Just to make it consistent with other buildvariants.

Reply:
Ah okay.